hs_core/templates/pages/genericresource.html

```diff
@@ -2701,32 +2977,10 @@ <h3 style="color: red" id="file-type-error"></h3>
             });
         });
 
-        $("#id-coverage-spatial input:radio").change(function () {
-            if ($(this).val() == "point") {
-                $("#div_id_north").show();
-                $("#div_id_east").show();
-                $("#div_id_elevation").show();
-                $("#div_id_northlimit").hide();
-                $("#div_id_eastlimit").hide();
-                $("#div_id_southlimit").hide();
-                $("#div_id_westlimit").hide();
-                $("#div_id_uplimit").hide();
-                $("#div_id_downlimit").hide();
-            }
-            else {
-                $("#div_id_north").hide();
-                $("#div_id_east").hide();
-                $("#div_id_elevation").hide();
-                $("#div_id_northlimit").show();
-                $("#div_id_eastlimit").show();
-                $("#div_id_southlimit").show();
-                $("#div_id_westlimit").show();
-                $("#div_id_uplimit").show();
-                $("#div_id_downlimit").show();
-            }
-        });
+
     });
     </script>
-{% endblock %}
 
+    {% endblock %}
+    <script async defer src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCG8oE0dNOkKju0PeMK63rwQR7gB7kdJ7w&libraries=drawing&callback=initMap"></script>
```

Comment:
@mauriel133 if you use google map api v3, you should not need google maps api key as stated here: http://stackoverflow.com/questions/5180648/do-i-really-need-the-google-maps-api-key with detailed documentation here: https://developers.google.com/maps/documentation/javascript/basics?csw=1#Welcome